src/modules/expression-evaluator/src/evaluateExpressionTestData.ts

```diff
@@ -888,28 +890,22 @@ testData.emailValidation = {
       children: [
         {
           operator: 'objectProperties',
-          children: [{ value: 'form.q3' }],
-        },
-        {
-          value: '^[A-Za-z0-9.]+@[A-Za-z0-9]+\\.[A-Za-z0-9.]+$',
+          children: ['form.q3'],
         },
+        '^[A-Za-z0-9.]+@[A-Za-z0-9]+\\.[A-Za-z0-9.]+$',
       ],
     },
     {
       operator: 'API',
       children: [
-        {
-          value: 'http://localhost:8080/check-unique',
-        },
-        {
-          value: ['type', 'value'],
-        },
-        { value: 'email' },
+        'http://localhost:8080/check-unique',
+        ['type', 'value'],
+        'email',
```

Comment:
It's a good idea to rewrite these using the simpler syntax, but we should make sure we leave some of the tests using the more verbose version so that gets tested too.

Reply:
Oh, sorry. I haven't though about that! I can add a new one specific to test that now

Reply:
Done: 59783cd

Reply:
Great, thanks. And it looks like there's still plenty of older tests using the old format for other operators, so that's all good.